docs/rest_api.md

```diff
@@ -173,6 +169,92 @@ Additionally, the `show_networks` api call may return the channel identifier
 Where documentation specifies that the network driver should choose a
 default channel, the VLAN drivers choose `vlan/native`.
 
+### list_networks
+
+`GET /networks`
+
+List all networks.
+
+Returns a JSON dictionary of dictionaries, indexed by the name of the network with interior dictionaries indexed by project and channel.
+
+The response must contain the following fields:
+
+* "network", the name of a network
+* "driver_id", the id of the network
+* "projects", a list of projects with access to the network or 'None' if network is public 
+
+Example Response:  
+	{
+		"netA": {
+			"driver_id": "101", 
+			"projects": ["qproj-01", qproj-02"]
+			},
+		"netB": {
+			"driver_id": "102", 
+			"projects": None}
+	}
+
+Authorization requirements:
+
+Only admins can list the networks.
+
+### list_network_attachments
+
+`GET /network/<network>/attachments`
+
+List all nodes that are attached to network <network>.
+    
+If optional argument 'project' is supplied will only list attached nodes belonging to the specified project.
+
+Returns a JSON dictionary of dictionaries with first level key being the name of the attached node and second level keys being:
+
+* "nic", the name of the nic on which the node is attached
+* "channel", the channel on which the attachment exists
+* "project", the name of the project which owns the attached node
+
+Example Response:  
+	{
+		"node1": {
+			 "nic": "nic1", 
+			 "channel" "vlan/native",
+			 "project": "projectA"
+			 }, 
+		"node2": {
+			 "nic": "nic2",
+			 "channel": "vlan/235",
+			 "project": "projectB"
+			 }
+	}
+
+Authorization requirements:
+
+Admins or the project that is the owner can list all attached nodes. Other projects can only list their own nodes.
```

Comment:
@henn, I hadn't realized you could do that; it should work since we're just calling `add_url_rule` ourselves (which is basically a trivial non-decorator version of `route`). Give it a shot.